procmonAnalyze.js

```diff
@@ -184,7 +206,7 @@ function parseReadDetail(detail) {
   }
 }
 
-async function drawData(data, diskify) {
+async function drawData(aData, diskify, isRefocused) {
```

Comment:
Let's not change arguments to hungarian notation (`aData`) here - moving `data` under `gOuterState` should be sufficient.

Reply:
Also, I don't think you need `isRefocused` here? Just set `gOuterState.data` to the value you pass into `drawData` from `readFileContents`.